purchases/src/main/kotlin/com/revenuecat/purchases/paywalls/components/common/Background.kt

```diff
@@ -8,7 +8,7 @@ import kotlinx.serialization.Serializable
 
 @InternalRevenueCatAPI
 @Serializable
-internal sealed interface Background {
+sealed interface Background {
```

Comment:
Hmm I'm wondering if we should namespace this one, since it's too generic... I believe we haven't been doing that for many of these, but I think we might be including a lot of API at the top level IMO...

Reply:
Right, but Kotlin has packages so namespacing isn't as needed as in Swift imo. The compiler makes sure we use the right type. Which kind of problems do you foresee?

Reply:
Mostly autocomplete related problems... As in, if someone is writing in their code a background related class, this would appear more easily in their autocomplete list I think? 

Naming collisions could also happen if they have a `Background` type and they would need to use import aliases (not that developers should ever need to use this in general so I'm less concerned about this.)

Reply:
Yeah I've recently had this issue with compose's Color vs android graphics color for example 😅... I think we can probably go ahead for now with this, but something to consider